packages/textlint/src/cli.ts

```diff
@@ -76,11 +76,26 @@ export const cli = {
             const descriptor = await loadDescriptor(currentOptions);
             Logger.log(JSON.stringify(descriptor, null, 4));
             return Promise.resolve(0);
+        } else if (currentOptions.mcp) {
+            const { server: mcpServer } = require("@textlint/mcp");
+            const { StdioServerTransport } = require("@modelcontextprotocol/sdk/server/stdio.js");
```

Comment:
What do you think of the possibility that `@textlint/mcp` itself could also provide a function to start the server?

```ts 
const { launchServer } = await import("@textlint/mcp"); 
const { closeServer } = launchServer({ /* additional options */ });

process.on("SIGINT", () => { 
  closeServer(); 
}); 
```